src/linux/classes/com/alibaba/wisp/engine/WispEventPump.java

```diff
@@ -242,23 +242,22 @@ private int epollWaitForWisp(int epfd, long pollArray, int arraySize, long timeo
                 return updated;
             }
         }
+        assert timeout != 0;
```

Comment:
WispConfiguration.MONOLITHIC_POLL == false
timeout ==0 and updated = 0, this assert could fail